src/main.c

```diff
@@ -247,6 +264,7 @@ static void import_existing_schedule( const char *data_file, const char *md5_fil
 
     if (0 != verify_md5_signatures(data_file, md5_file)) {
         debug_error("import_existing_schedule() data or md5 corruption\n");
+	set_aker_metrics(MEC, 1, 1);
```

Comment:
Try to match the code format (spaces vs tabs) that is present.  It helps keep the code more consistent.